pythia/modules/metrics.py

```diff
@@ -49,6 +49,7 @@ def calculate(self, sample_list, model_output):
 import torch
 
 from pythia.common.registry import registry
+from nltk.translate.bleu_score import corpus_bleu
```

Comment:
Let's not import it here. Import it only when your metric is initialized. This way this doesn't become a hard dependency.